packages/frontend/src/app/login/page.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { LoginButton } from "@/components/button/LoginButton";
+import { TextEditor } from "@/components/common/TextEditor";
+
+export default function LoginScene() {
+  return (
+    <main className="flex h-[100vh] w-[100vw] justify-center bg-gray-800">
+      <div className="flex w-[50%] min-w-[400px] max-w-[600px] flex-col">
```

Comment:
```suggestion
      <div className="flex w-[80%] min-w-[300px] max-w-[800px] flex-col justify-center gap-6">
```

縦幅は個別でマージン設定の必要はなく、 flexbox での調整で十分となります